docker-run.sh

```diff
@@ -21,6 +21,16 @@ replace_or_delete_in_index () {
   fi
 }
 
+if [ "${BASE_URL}" ]; then
+  NGINX_WITH_BASE_URL="${NGINX_ROOT}${BASE_URL}"
+
+  mkdir -p ${NGINX_WITH_BASE_URL}
+  mv ${NGINX_ROOT}/*.* ${NGINX_WITH_BASE_URL}/
```

Comment:
How does this behave when the container is run multiple times? I think it would nest the folder each time, i.e. `/swagger` then `/swagger/swagger` and so on.

Reply:
The state of the container is reset when you kill it, so each time you run the container, this script runs against a fresh file system, with all of the UI files in `/usr/share/nginx/html`.

I verified this locally by running it multiple times with the same base url, and a few times with the url switched back and forth to something else.  Each time it worked as expected.